Dockerfile

```diff
@@ -584,6 +590,15 @@ RUN \
   cd opencore-amr-* && ./configure --enable-static --disable-shared && \
   make -j$(nproc) install
 
+RUN \
+  git clone "$LIBGSM_URL" && \
+  cd libgsm && git checkout $LIBGSM_COMMIT && \
+  # Makefile is garbage, hence use specific compile arguments and flags
+  SRC=$(echo src/*.c) && CFLAGS='-c -ansi -pedantic -O3 -static-libgcc -fno-strict-overflow -fstack-protector-all -fPIE -s -DNeedFunctionPrototypes=1 -Wall -Wno-comment -DSASR -DWAV49 -DNDEBUG -I./inc -Wl,-z,relro,-z,now' && \
+  gcc ${CFLAGS} ${SRC} && mv *.o ./src && ar cr libgsm.a ${SRC/\.c/\.o} && ranlib libgsm.a && \
+  cp inc/*.h /usr/include && \
```

Comment:
The build error is ”fatal error: gsm/gsm.h: No such file or directory” should be in include/gsm?